helix-term/src/ui/picker.rs

```diff
@@ -984,31 +985,58 @@ impl<T: 'static + Send + Sync, D: 'static + Send + Sync> Picker<T, D> {
             );
         }
     }
+
+    fn get_layout(&self, area: Rect) -> (bool, bool) {
+        let render_preview = self.show_preview
+            && self.file_fn.is_some()
+            && area.width >= MIN_AREA_WIDTH_FOR_PREVIEW
+            && area.height >= MIN_AREA_HEIGHT_FOR_PREVIEW;
```

Comment:
Found a small bug, playing with your patch locally (great work btw!)

I think this should be an OR condition rather than AND, so 
```
(area.width >= MIN_AREA_WIDTH_FOR_PREVIEW || area.height >= MIN_AREA_HEIGHT_FOR_PREVIEW)
```

Reply:
Fixed by 4ff2160